modules/tracing/mule-tracer-configuration-impl/src/main/java/org/mule/runtime/tracing/level/impl/config/FileTracingLevelConfiguration.java

```diff
@@ -53,30 +54,45 @@ public FileTracingLevelConfiguration(MuleContext muleContext) {
   @Override
   public TracingLevel getTracingLevel() {
     Properties properties = getTracingLevelProperties();
```

Comment:
if possible try to read the file only once (not every time you invoke the trace level), by using an initialise or make it lazy, or do not retrieve the file once you have it. This was from the previous implementation.

Reply:
I made some changes to achieve this. Please review them. Thanks!